config/config/sc-config.yaml

```diff
@@ -11,6 +11,11 @@ global:
   clusterDns: "10.43.0.10"
 
 objectStorage:
+  # Options are "s3", "gcs", or "none"
+  # If "none", remember to disable features that depend on object storage:
+  #   all backups (velero, harbor, influxdb, elasticsearch), sc logs (fluentd)
+  #   Also set harbor persistence to "filesystem" or "swift"
```

Comment:
A bit counter-intuitive imo having `objectStorage: none`, but saying that you can use objectstorage (swift) for harbor persistence :P

Reply:
I agree, but do you think I should prevent people from using swift while we have `objectStorage: none`?

Reply:
No, not nessecarily as we do not support swift for backups right?